pyproject.toml

```diff
@@ -1,12 +1,101 @@
 [build-system]
-requires = [
-    "setuptools>=42",
-    "wheel"
+requires = ["flit_core >=3.4,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "oteapi-dlite"
+authors = [
+    {name = "SINTEF", email = "Team4.0@SINTEF.no"},
+]
+description = "DLite plugin for Open Translation Environment (OTE) API."
+readme = "README.md"
+license = {file = "LICENSE"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "License :: OSI Approved :: MIT License",
 ]
-build-backend = "setuptools.build_meta"
+keywords = ["OTE", "OTEAPI", "LITE", "OTEAPI-DLITE"]
+requires-python = ">=3.9"
+dynamic = ["version"]
+
+dependencies = [
+    # Core dependencies
+    "DLite-Python>=0.4.5,<1.0",
+    "numpy>=1.21,<1.27.0",
+    "oteapi-core>=0.6.0,<0.6.2",
+    "pydantic~=2.5",
+    "Pillow>=9.0.1,<11",
+    "tripper>=0.1.2",
+]
+
+[project.optional-dependencies]
+docs = [
+    "mike~=2.1",
+    "mkdocs>=1.5.3,<1.7",
+    "mkdocs-awesome-pages-plugin~=2.9",
+    "mkdocs-material>=9.5.5,<10",
+    "mkdocstrings[python]~=0.25.1",
```

Comment:
Shouldn't these be replaced with `==`?

Reply:
~= means this version or higher minor versions. It is fine

Reply:
Not that important, but for generating documentation I think it is safer to be strict. But lets not spend  time on it now...